velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -836,7 +836,7 @@ PlanBuilder& PlanBuilder::partitionedOutput(
       nextPlanNodeId(),
       exprs(keys),
       numPartitions,
-      false,
+      core::PartitionedOutputNode::Kind::kPartitioned,
```

Comment:
We shall pass this as partitionedOutput input?

Reply:
Can you elaborate your question?

Reply:
Ignore my question.